schemas/pioneer/heatmap/heatmap.1.parquetmr.txt

```diff
@@ -0,0 +1,13 @@
+message pioneer_heatmap {
+  required binary user (UTF8);
+  required group sessions (LIST) {
+    repeated group list {
+      required group element {
+        optional int64 duration;
+        required int64 start_time;
+        optional binary tab_id (UTF8);
+        required binary url (UTF8);
+      }
+    }
+  }
+}
```

Comment:
Do you want to include any metadata? e.g.

        required group metadata {
            required int64  Timestamp;
            required binary documentId (UTF8);
            optional binary geoCountry (UTF8);
            optional binary geoCity (UTF8);
        }